bootstrap/playbook.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Playbook for initialising new hosts for OME developers
```

Comment:
It's not just that though- it includes resizing LVs and optionally network config to make it easier for OME devs to use non-hardware ansible roles.

Reply:
Let's try to express that such that it's clear this could be valuable for others than just the OME team. "Playbook which runs the necessary root-level steps so that a host can be managed by others"?

Reply:
Updated with your suggested wording, though the playbook still makes assumptions about the hardware (e.g. it assumes a volgroup that needs expanding).